be/src/olap/tablet_meta.h

```diff
@@ -285,6 +289,38 @@ inline void TabletMeta::set_cumulative_layer_point(int64_t new_point) {
     _cumulative_layer_point = new_point;
 }
 
+inline bool TabletMeta::contains_delete(const Version& version) const {
+    for (auto& rs : _rs_metas) {
```

Comment:
I am not sure if rs in _rs_metas are ordered by version?

Reply:
no need to order by version, if contains delete version this tablet will not split by segmnent